torch_xla/csrc/runtime/pjrt_registry.cc

```diff
@@ -43,6 +43,29 @@ std::optional<std::string> GetPjRtPluginPath(const std::string& device_type) {
                                             : std::nullopt;
 }
 
+std::unique_ptr<XlaCoordinator> SetKeyValueCallback(
+    int global_process_rank, int global_world_size,
+    std::unique_ptr<XlaCoordinator> coordinator,
```

Comment:
Why do we need the coordinator as input here?

Reply:
We need it get the DistributedRuntimeClient and later create the `kv_store` below

Reply:
It looks like it's being recreated on L60 - should we just make this function return the new value?

Reply:
done